### PR TITLE
docs: add github.com/ev3go/ev3 to other languages

### PR DESCRIPTION
**Pull request by kortschak**

### CODE REVIEW

File: docs/libraries.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ libraries may be outdated due to the fast development cycle of ev3dev.
 
 * Extra languages:
     * [Google Go](https://github.com/ldmberman/GoEV3) updated for ev3dev-jessie by @ldmberman, [original](https://github.com/mattrajca/GoEV3) by @mattrajca
```
Comment (kortschak) [inline docs/libraries.md]: This should be changed, `s/Google Go/Go/` according to name usage by the Go community.

Comment (dlech) [inline docs/libraries.md]: Sounds good to me. Would you like to add that to your pull request?

Comment (kortschak) [inline docs/libraries.md]: Done

**Comment by dlech** [comment]
Thanks!

**Pull request merged**